pkg/router/controller/status.go

```diff
@@ -97,10 +97,16 @@ func findOrCreateIngress(route *routeapi.Route, name string) (_ *routeapi.RouteI
 // setIngressCondition records the condition on the ingress, returning true if the ingress was changed and
 // false if no modification was made.
 func setIngressCondition(ingress *routeapi.RouteIngress, condition routeapi.RouteIngressCondition) bool {
-	for _, existing := range ingress.Conditions {
+	for i, existing := range ingress.Conditions {
 		// ensures that the comparison is based on the actual value, not the time
 		existing.LastTransitionTime = condition.LastTransitionTime
 		if existing == condition {
+			// This will always be the case if we're receiving an update on the host
+			// value (or the like), since findOrCreateIngress sets that for us.  We
+			// still need to set the last-touched time so that others can tell we've
+			// modified this Ingress value
+			now := nowFn()
+			ingress.Conditions[i].LastTransitionTime = &now
```

Comment:
I think it actually should stay as `False` -- it's only used in the rejection path, and it's actually in a path where we don't want to update if that's the only thing that would be changed, AFAICT, but I'll double check in the morning when I'm more awake ;-).